Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -644,6 +733,10 @@ public static void GetDeactivatedAssetNotification(Action<LootLockerDeactivatedA
             LootLockerAPIManager.GetDeactivatedAssetNotification(onComplete);
         }
 
+        /// <summary>
+        /// 5 minutes after calling this endpoint you should issue a call to the Player Asset Notifications call to get the results of the migrated DLC, if any.
+        /// </summary>
+        /// <param name="onComplete">onComplete Action</param>
```

Comment:
This returns when the request has been accepted I assume. Is there any method to check for progress?

Reply:
Unfortunately not. It is either done or not done.